ui/components/src/Table/Table.tsx

```diff
@@ -22,6 +30,12 @@ const DEFAULT_GET_ROW_ID = (data: unknown, index: number) => {
   return `${index}`;
 };
 
+// Setting these defaults one enables them to be consistent across renders instead
+// of being recreated every time, which can be important for perf because react
+// does not do deep equality checking for objects and arrays.
+const DEFAULT_ROW_SELECTION: NonNullable<TableProps<unknown>['rowSelection']> = {};
+const DEFAULT_SORTING: NonNullable<TableProps<unknown>['sorting']> = [];
```

Comment:
Any idea why this is an array? Can you sort more than one column at a time?

Reply:
We don't support multisort right now, but we could in the future, so starting with it as an array would allow us to add that without changing the type.